src/config/media/pipewire.md

```diff
@@ -0,0 +1,87 @@
+# PipeWire
+
+To use PipeWire, install the `pipewire` package.
+
+PipeWire should be started as a user. Use the autostarting mechanism of your
+desktop environment.
+
+If using [startx](../graphical-session/xorg.md#startx), add the following line
+in your `.xinitrc` file:
+
+```
+pipewire &
+```
+
+If using [Sway](https://swaywm.org/), start it from the configuration file:
+
+```
+exec pipewire
+```
+
+## PulseAudio replacement
+
+PipeWire can start a PulseAudio server. While it's possible to modify
+`/etc/pipewire/pipewire.conf` for auto-starting the PulseAudio server, it's
+recommended to start it from the same place where you start PipeWire by
+executing `pipewire-pulse`. This keeps PipeWire's configuration file unmodified
+for smoother future upgrades.
+
+#### Example for `.xinitrc`
+
+```
+pipewire &
+pipewire-pulse &
+```
+
+#### Example for Sway
+
+```
+exec pipewire
+exec pipewire-pulse
+```
+
+After setting up `pipewire-pulse`, make sure that the PulseAudio service is
+[disabled](../services/index.md#disabling-services) and reboot.
```

Comment:
More important is probably to disable PulseAudio auto activation (I can add a section on that), because most people shouldn't be using the service already.

Reply:
What do you mean by disabling auto activation?
I think pulseaudio should not start automatically if the pipewire-pulse server is running.

Reply:
Pulseaudio can start automatically before pipewire-pulse is ran which creates a conflict between them.